src/globus_cli/commands/timer/create/transfer.py

```diff
@@ -9,13 +9,18 @@
 import globus_sdk
 
 from globus_cli.endpointish import Endpointish
-from globus_cli.login_manager import LoginManager
+from globus_cli.login_manager import (
+    LoginManager,
+    is_client_login,
+    read_well_known_config,
+)
 from globus_cli.parsing import (
     ENDPOINT_PLUS_OPTPATH,
     TimedeltaType,
     command,
     encrypt_data_option,
     fail_on_quota_errors_option,
+    no_local_server_option,
```

Comment:
```suggestion
```

Need to remove this and the usage. It was in an earlier version of the command but is not what we want to be doing.